conjure-runtime/src/blocking/request.rs

```diff
@@ -129,3 +136,35 @@ impl<'a> RequestBuilder<'a> {
         }
     }
 }
+
+#[pin_project]
+struct ContextFuture<F> {
```

Comment:
is there any way you can make use of bind? (https://github.com/palantir/rust-zipkin/blob/master/zipkin/src/open_span.rs#L184)

Reply:
Not that one, since we don't have ahold of the current `OpenSpan`. Something like this should end up in the zipkin crate, but I'm not sure exactly what it should look like.

Reply:
Yeah I guess in java land we have this fairly unwieldy utility class (https://github.com/palantir/tracing-java/blob/develop/tracing/src/main/java/com/palantir/tracing/Tracers.java) which just provides a bunch of `wrap` methods